detectree2/models/train.py

```diff
@@ -391,6 +391,61 @@ def build_train_loader(cls, cfg):
         ),
     )
 
+  # def build_optimizer(cls, cfg, model):
```

Comment:
using adam optimizer instead of SGD. While SGD is updated recently so this function is commented out.